code/modules/xenoarchaeology/xenoartifact_traits.dm

```diff
@@ -0,0 +1,1011 @@
+	if(istype(item, /obj/item/multitool))
+		to_chat(user, "<span class='info'>The [item.name] displays an overcharge reading of [charges/3].</span>") 
+		return TRUE
+	..()
+
+/datum/xenoartifact_trait/minor/dense //Rather large, quite gigantic, particularly big
+	desc = "Dense"
+	label_desc = "Dense: The Artifact is dense and cannot be easily lifted but, the design has a slightly higher reaction rate."
+	blacklist_traits = list(/datum/xenoartifact_trait/minor/wearable, /datum/xenoartifact_trait/minor/sharp, /datum/xenoartifact_trait/minor/light, /datum/xenoartifact_trait/minor/heavy, /datum/xenoartifact_trait/minor/blocking)
+
+/datum/xenoartifact_trait/minor/dense/on_init(obj/item/xenoartifact/X)
+	X.density = TRUE
+	X.interaction_flags_atom = INTERACT_ATOM_ATTACK_HAND
+	X.interaction_flags_item = INTERACT_ATOM_ATTACK_HAND
+
+/datum/xenoartifact_trait/minor/sharp
+	desc = "Sharp"
+	label_desc = "Sharp: The Artifact is shaped into a fine point. Perfect for popping balloons."
+	blacklist_traits = list(/datum/xenoartifact_trait/minor/dense)
+
+/datum/xenoartifact_trait/minor/sharp/on_touch(obj/item/xenoartifact/X, mob/user)
+	to_chat(user, "<span class='notice'>The [X.name] feels sharp.</span>")
+	return TRUE
+
+/datum/xenoartifact_trait/minor/sharp/on_init(obj/item/xenoartifact/X)
+	X.sharpness = IS_SHARP_ACCURATE
+	X.force = X.charge_req*0.15
+	X.attack_verb = list("cleaved", "slashed", "stabbed", "sliced", "tore", "ripped", "diced", "cut")
+	X.attack_weight = 2
+	X.armour_penetration = 5
+
+/datum/xenoartifact_trait/minor/radioactive
+	label_name = "Radioactive"
+	label_desc = "Radioactive: The Artifact Emmits harmful particles when a reaction takes place."
+
+/datum/xenoartifact_trait/minor/radioactive/on_init(obj/item/xenoartifact/X)
+	X.AddComponent(/datum/component/radioactive, 25)
+
+/datum/xenoartifact_trait/minor/radioactive/on_item(obj/item/xenoartifact/X, atom/user, atom/item)
+	if(istype(item, /obj/item/geiger_counter))
+		to_chat(user, "<span class='notice'>The [X.name] has residual radioactive decay features.</span>")
+		return TRUE
+	..()
+
+/datum/xenoartifact_trait/minor/radioactive/on_touch(obj/item/xenoartifact/X, mob/user)
+	to_chat(user, "<span class='notice'>You feel pins and needles after touching the [X.name].</span>")
+	return TRUE
+
+/datum/xenoartifact_trait/minor/radioactive/activate(obj/item/xenoartifact/X)
+	X.AddComponent(/datum/component/radioactive, 80)
+
+/datum/xenoartifact_trait/minor/cooler //Faster cooldowns
+	desc = "Frosted"
+	label_desc = "Frosted: The Artifact has the unique property of actively cooling itself. This also seems to reduce time between uses."
+
+/datum/xenoartifact_trait/minor/cooler/on_touch(obj/item/xenoartifact/X, mob/user)
+	to_chat(user, "<span class='notice'>The [X.name] feels cold.</span>")
+	return TRUE
+
+/datum/xenoartifact_trait/minor/cooler/on_init(obj/item/xenoartifact/X)
+	X?.cooldown = 4 SECONDS //Might revisit the value.
```

Comment:
This probably shouldn't set it to 4 always, since there are some that have cooldowns for a reason

Reply:
This is base cooldown. Traits like time-stop that have a longer cooldown add to this, through ``cooldownmod``. This brings the base down from ``8 SECONDS`` to ``4 SECONDS``, so a minute cooldown would be 56 seconds. 
This is just useful for traits that don't have extra cooldowns, like lamp & shade.